container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -340,10 +353,22 @@ function run_rpm_tempest {
     # List Tempest packages
     rpm -qa | grep tempest
 
+    tempest cleanup --init-saved-state
+
+    if [[ ! ${#TEMPEST_EXTRA_IMAGES_NAME[@]} -eq 0 ]]; then
+        upload_extra_images
+    fi
```

Comment:
I know this code already existed here and this change just moves it around. But since there are other changes required we can simplify [1] this as:
```
# No if-statement
upload_extra_images
```

[1] https://github.com/openstack-k8s-operators/tcib/pull/208/files#diff-191bc7762340f2539b035115667e34a49916229d47d776aca4a2c457f694e35cR228